js/compressed-token/src/idl/light_compressed_token.ts

```diff
@@ -5,10 +5,9 @@ export type LightCompressedToken = {
         {
             name: 'createTokenPool';
             docs: [
-                'This instruction expects a mint account to be created in a separate',
-                'token program instruction with token authority as mint authority. This',
-                'instruction creates a token pool account for that mint owned by token',
-                'authority.',
+                'This instruction creates a token pool for a given mint. Every spl mint',
+                'can have one token pool. When a token is compressed the compressed',
+                'tokens are transferrred to the token pool.',
```

Comment:
imo it's easier to understand if we say something like this:
```suggestion
                'can have one token pool. When a token is compressed the SPL',
                'tokens are transferrred to the token pool, and their compressed equivalent is minted into a Merkle tree.',
```